src/modules/cmdpal/Microsoft.CmdPal.UI/ExtViews/ListPage.xaml.cs

```diff
@@ -121,6 +122,18 @@ private void ItemsList_DoubleTapped(object sender, DoubleTappedRoutedEventArgs e
         }
     }
 
+    private void GroupedItemsList_DoubleTapped(object sender, DoubleTappedRoutedEventArgs e)
+    {
+        if (GroupedItemsList.SelectedItem is ListItemViewModel vm)
+        {
+            var settings = App.Current.Services.GetService<SettingsModel>()!;
+            if (!settings.SingleClickActivates)
+            {
+                ViewModel?.InvokeItemCommand.Execute(vm);
+            }
+        }
+    }
```

Comment:
Future note: could use a custom XAML behavior for this so it can be re-used across different page/item types.